.github/workflows/test.yml

```diff
@@ -35,22 +35,28 @@ jobs:
             - py36-dj22
             - py36-dj30
             - py36-dj31
```

Comment:
Since Django 3.2 is an LTS release, I'd be okay dropping 3.0 and 3.1 from our test matrix. See following suggestions.

```suggestion
```